### ISSUE TITLE
Quick question - Path to upgrade harbor from 1.10.0 to 2.2.0

### ISSUE THREAD

**Issue by Grounz** [goharbor/harbor#14931]
Hi,

I want upgrade my harbor stack from 1.10.0 to 2.2.0. I have read the doc and i undesterdand i can do it directy withou pass by 2.0.0 version or 2.1.0, it's true ?

Because in 2.0.0 i can see above description (i found that here https://github.com/goharbor/harbor/releases/tag/v2.0.0):

```
Change in installation and upgrade process:
For installation based on docker-compose, the default configuration file in the installation package is renamed to harbor.yml.tmpl. You must set the values, rename it to harbor.yml before running the installation script.
The migrator image is removed and now the migrate is a sub-command of prepare. More details please refer to: https://goharbor.io/docs/2.0.0/administration/upgrade/
```

But in 2.2.0 or in 2.1.0, this descriptions is not present, that's why i ask  the question. Or this description is also true for an upgrade from 1.10.0 to 2.2.0 direct ?

Maybe i must only upgrade from 1.10.0 to 2.0.0, then 2.0.0 to 2.1.0 and 2.1.0 to 2.2.0 ?

**Comment by cdchris12** [comment]
We just did an upgrade from `v1.10.3` to `v2.1.5` using the Harbor-Helm chart, and everything was upgraded for us automatically by the `core` container when it started up. The only thing we had to change was to set the `core` container's `livenessProbe` `periodSeconds` property to `28800` in order to give the `core` container time to do all the work it needed to do. 

It's important to note that our Harbor was taken offline during this process, as the `core` container will not process any requests until it has entirely completed the upgrade process. Our 19TB of images stored in s3 took about 3 hours to complete the image abstraction process which comes with the move from Harbor v1 to Harbor v2.
